openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/settings/RubyGemsSettings.vue

```diff
@@ -55,7 +55,7 @@ export default {
   mixins: [Settings],
   data() {
     return {
-      rubygemsUrl: 'https://rubygems.org',
+      rubygemsUrl: process.env.RUBYGEMS_URL,
```

Comment:
Client side web browser code doesn't have environment variables. This is nodejs style backend code that won't work in the browser.  This is only setting the default value and should get overridden with actual value if it has been set.

Reply:
Reverted this back.